apps/ledger-live-desktop/tests/models/SettingsPage.ts

```diff
@@ -17,6 +24,18 @@ export class SettingsPage {
     this.experimentalTab = page.locator("data-test-id=settings-experimental-tab");
     this.experimentalDevModeToggle = page.locator("data-test-id=MANAGER_DEV_MODE-button");
     this.carouselSwitchButton = page.locator("data-test-id=settings-carousel-switch-button");
+    this.counterValueSelector = page.locator(".select__value-container").first();
+    this.counterValueSearchBar = page.locator('[placeholder="Search"]');
+    this.counterValuedropdownChoiceEuro = page.locator(
+      "#react-select-2-option-15 >> text=Euro - EUR",
+    );
+    this.languageSelector = page.locator(
+      "div:nth-child(2) > .sc-dkzDqf.uXYMl > .css-198krsd-container > .select__control > .select__value-container",
+    );
+    this.themeSelector = page.locator(
+      "div:nth-child(4) > .sc-dkzDqf.uXYMl > .css-198krsd-container > .select__control > .select__value-container",
+    );
+    this.themeChoiceLight = page.locator("#react-select-5-option-1");
```

Comment:
Again this seems brittle. I don't know much about `react-select` but it seems dynamic from a package that we don't control, so if it changes on their end, it will break the tests. Try to add a `data-test-id`